js/views/userPage/UserPageHome.js

```diff
@@ -0,0 +1,77 @@
+import loadTemplate from '../../utils/loadTemplate';
+import { View } from 'backbone';
+import app from '../../app';
+
+export default class extends View {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    this.ownPage = options.ownPage;
+    this.isModerator = true; // TODO set if this user is a moderator
```

Comment:
If we're going to give the subviews this.model, they could get isModerator right from the model.